OsmAnd-java/src/main/java/net/osmand/binary/BinaryMapIndexReader.java

```diff
@@ -1784,6 +1794,7 @@ public static class SearchRequest<T> {
 		SearchFilter searchFilter = null;
 
 		SearchPoiTypeFilter poiTypeFilter = null;
+		SearchPoiSubcategoryFilter subcategoryFilter;
```

Comment:
it's not subcategory. Category is key for Amenity, subcategory would be "cinema", ... however it's called poitype.

So it's definitely bad name for poi Additional tags